types/node/index.d.ts

```diff
@@ -1852,9 +1852,22 @@ declare module "url" {
         hash?: string;
         path?: string;
     }
+    
+    export interface FormatOptions {
```

Comment:
I think the name `FormatOptions` is not that good as in the end this reflects the object to format not the options configuring formatting. Not sure about the best name here but maybe something like `UrlObject` as in the node docu?